ASM/c/kaleido_scope.h

```diff
@@ -0,0 +1,73 @@
+#pragma once
```

Comment:
Does out compiler support `#pragma once`?

Reply:
We use a modern version of `gcc` so it should.